### DISCUSSION TITLE
How to add "Save Metadata to Images" to config.txt

### DISCUSSION THREAD

**Discussion by bblackmoor** [lllyasviel/Fooocus#2516] [Q&A] [answered]
Is it possible to add the "**Save Metadata to Images**" setting to **config.txt**?

If so, how, please? I would like to make it so that this is the default for me.

Thank you.

**Comment by mashb1t** [comment, marked as answer]
Yes, this is possible. Please copy the keys over from the config_modification_tutorial.txt

for me:
```
    ...
    ],
    "default_save_metadata_to_images": true,
    "default_metadata_scheme": "fooocus",
    "metadata_created_by": "mashb1t"
}
```

Keep in mind that you must not have a comma at the last line.

**Reply by bblackmoor** [reply]
Thank you!

**Reply by mikecastrodemaria** [reply]
Many thanks !!!

**Reply by Emerson-Eberle** [reply]
thanks
